CanvasCore/CanvasCore/Helm/HelmConstants.swift

```diff
@@ -31,7 +31,7 @@ struct PropKeys {
     static let navBarStyle = "navBarStyle"
     static let navBarColor = "navBarColor"
     static let navBarHidden = "navBarHidden"
-    static let navBarImage = "navBarImage"
+    static let navBarLogo = "navBarLogo"
```

Comment:
anything with `/image/i` gets converted to an image. This needs to be a boolean.